packages/klighd-core/src/options/components/option-inputs.tsx

```diff
@@ -56,26 +56,32 @@ interface ChoiceOptionProps extends BaseProps<string> {
     availableValuesLabels?: string[];
 }
 
+/** This Function calls the onChange method, so that a click cause an active change */
+function getCurrentValue(e: Event, props: ChoiceOptionProps) {
+    if (e.target == null) {
+        return
+    }
+    if (!(e.target instanceof HTMLSelectElement)) {
+        return
+    }
+    const val = e.target.value
+    return props.onChange(val)
```

Comment:
such small otherwise unused expressions can be included in the method call directly, i.e. `props.onChange(e.target.value)`.
remove the return here, a void method does not need to conclude with a return in the end and should also not return some value.